server/repository/src/db_diesel/master_list_line_row.rs

```diff
@@ -60,4 +63,21 @@ impl<'a> MasterListLineRowRepository<'a> {
             .first(&self.connection.connection)?;
         Ok(result)
     }
+
+    pub fn find_one_by_id_option(
```

Comment:
Added this due to check_records_against_database being cumbersome, see PR comment